app/controllers/valid_attachment_types_controller.rb

```diff
@@ -0,0 +1,8 @@
+class ValidAttachmentTypesController < ApplicationController
+  def index
+    skip_policy_scope
+    skip_authorization
+
+    render json: [RadCommon::VALID_ATTACHMENT_TYPES], status: :ok
```

Comment:
maybe expand this endpoint to also include file size limit of 100mb, which is going to be shared by the template helper that checks